infrastructure/pom.xml

```diff
@@ -90,6 +90,43 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>compatibility</id>
```

Comment:
we could just add a tag into docker profile, no need for another profile. If you'd like to merge these two profiles then please create another pr